.github/workflows/test.yml

```diff
@@ -36,7 +36,17 @@ jobs:
       - name: Run build
         run: npm run build
 
-      - name: Run all tests
+      - name: Run all tests (without coverage)
+        run: npm run test:ci -- --coverage=false
```

Comment:
Since this passes in the current `main`, I went back to the failing `mock-jwks` version with `git checkout cb1104e` and then ran `npm test:ci`, verified it exited `0` and then ran `npx jest --config=jest.config.ci.js --runInBand --coverage=false` (the equivalent of this line here) and verified it exited `1`.